browser/extensions/updater/brave_update_client_config.cc

```diff
@@ -243,7 +244,7 @@ BraveUpdateClientConfig::GetProtocolHandlerFactory() const {
 
 update_client::RecoveryCRXElevator
 BraveUpdateClientConfig::GetRecoveryCRXElevator() const {
-#if defined(GOOGLE_CHROME_BUILD) && defined(OS_WIN)
+#if BUILDFLAG(GOOGLE_CHROME_BRANDING) && defined(OS_WIN)
```

Comment:
same as below, it makes no sense to have a GOOGLE_CHROME_ANYTHING buildflag in a brave file

Reply:
should this just be OFFICIAL_BUILD or not exist at all?

Reply:
Fixed.